manga_ocr_dev/training/get_model.py

```diff
@@ -48,7 +47,7 @@ def get_model(encoder_name, decoder_name, max_length, num_decoder_layers=None):
 
         decoder_config.num_hidden_layers = num_decoder_layers
 
-    config = VisionEncoderDecoderConfig.from_encoder_decoder_configs(encoder.config, decoder.config)
+    config = VisionEncoderDecoderConfig.from_encoder_decoder_configs(encoder_config, decoder_config)
```

Comment:
Probably this is a existing bug ? 

Otherwise, the saved config file and model will be different, i.e.: in the config for decoder, there are more layers than the model. When load the model for inference, those layers will be randomly initialized.